src/components/FlowRunFilteredList.vue

```diff
@@ -40,7 +40,7 @@
   const props = defineProps<{
     flowRunFilter: FlowRunsFilter,
     states?: PrefectStateNames[],
-    disabled?: boolean,
+    disableDeletion?: boolean,
```

Comment:
Can we use `selectable` here as well?

Reply:
Ya know what, I don't actually see an instance where this is being disabled anywhere. The `can` check in here is relied on. I'll just remove it.